src/Controls/src/Core/BindableProperty.cs

```diff
@@ -229,10 +229,9 @@ internal bool TryConvert(ref object value)
 			if (returnType.IsAssignableFrom(valueType))
 				return true;
 
-			var cast = returnType.GetImplicitConversionOperator(fromType: valueType, toType: returnType) ?? valueType.GetImplicitConversionOperator(fromType: valueType, toType: returnType);
-			if (cast != null)
+			if (value.TryConvertValue(toType: returnType, out var convertedValue))
```

Comment:
this might break user code

Reply:
Yes, I think we probably need a feature switch for implicit operators. I think if the switch was toggled for NativeAOT, we could make all `GetImplicitConversionOperator` and similar methods return `null`?

I don't think this case we can make it throw (like usual feature switches). We have to just make it *not find* the operator under NativeAOT?

Reply:
Yes, my idea was to add a feature switch that would turn on "backwards compatibility mode" and that in that mode `TryConvertValue` would use the implicit operators exactly the same way it does today: https://github.com/dotnet/maui/pull/19992/files#diff-91f94858941625b682048774f0e2d281a07d08ee17c25698df94936beac16f27R263-R267